darwin/importer/formats/nifti.py

```diff
@@ -531,9 +546,13 @@ def process_nifti(
     img = correct_nifti_header_if_necessary(input_data)
     orig_ax_codes = nib.orientations.aff2axcodes(img.affine)
     orig_ornt = nib.orientations.axcodes2ornt(orig_ax_codes)
+    if remote_file_path in remote_files_that_require_legacy_scaling:
+        slot_affine_map = remote_files_that_require_legacy_scaling[remote_file_path]
+        affine = slot_affine_map[next(iter(slot_affine_map))]  # Take the 1st slot
```

Comment:
There was work done in `MED-13` to allow export of `NifTI` annotations from multi-slotted items, but it has never been possible to import `NifTI` annotations to multi-slotted items. We therefore take the 1st slot in this case